money.go

```diff
@@ -198,26 +198,56 @@ func (m *Money) Negative() *Money {
 }
 
 // Add returns new Money struct with value representing sum of Self and Other Money.
-func (m *Money) Add(om *Money) (*Money, error) {
-	if err := m.assertSameCurrency(om); err != nil {
-		return nil, err
+func (m *Money) Add(om ...*Money) (*Money, error) {
```

Comment:
It's typically to add an "s" to the name to denote an array of:
```suggestion
func (m *Money) Add(ms ...*Money) (*Money, error) {
```